src/SymbolServer.jl

```diff
@@ -27,6 +27,17 @@ mutable struct SymbolServerInstance
     end
 end
 
+function remove_non_general_pkgs!(pkgs)
```

Comment:
Can make this a more general filter function that also filter out packages with `nothing` as UUID, no point in trying  those URLs either.

Reply:
I added this, but AFAICT because pkgs comes from the manifest.deps, which is a `Dict{Base.UUID, Pkg.Types.PackageEntry}()` it's not possible for the UUID to be nothing. But there is support here for <1.1 julia, which I struggle to follow (and question why it's still supported..) so I'll leave these checks in

Reply:
I just know I have seen URLs with `nothing` in the LS.jl logs, even on recent Julias. Perhaps for stdlibs?

Reply:
I think I have too, but I can't figure out how a pkg without uuid could end up in `pkgs` here. @davidanthoff any idea? It'd be good to test it

Reply:
Seems like it was the `tree_hash` being nothing that we were seeing, i.e. stdlibs, so I added filtering and testss